ddtrace/internal/datadog/profiling/ddup/_ddup.pyx

```diff
@@ -234,7 +235,6 @@ cdef class SampleHandle:
     def push_task_id(self, task_id: Optional[int]) -> None:
         if self.ptr is not NULL:
             if task_id is not None:
-                ddup_push_task_id(self.ptr, task_id)
```

Comment:
Why is this line deleted?

Reply:
Sorry, it's a weird artifact of the merge.  I actually replaced it with the line below (line 242)
```
ddup_push_task_id(self.ptr, clamp_to_int64_unsigned(task_id))
```